### PR TITLE
Add SigningMechanism.ImportKeysFromBytes

### PR DESCRIPTION
**Pull request by mtrmac**
This will be needed for verification against specified public keys.

Also rerun `hack/vendor.sh` to pick up import support from `github.com/mtrmac/gpgme` .

### CODE REVIEW

File: signature/mechanism.go [hunk 1]
```diff
@@ -13,6 +13,10 @@ import (
 // FIXME: Eventually expand on keyIdentity (namespace them between mechanisms to
 // eliminate ambiguities, support CA signatures and perhaps other key properties)
 type SigningMechanism interface {
+	// ImportKeysFromBlob imports public keys from the supplied blob and returns their identities.
+	// The blob is assumed to have an appropriate format (the caller is expected to know which one).
+	// NOTE: This may modify long-term state (e.g. key storage in a directory underlying the mechanism).
+	ImportKeysFromBlob(blob []byte) ([]string, error)
```
Comment (runcom) [inline signature/mechanism.go]: Can we call this `FromBytes` (I've seen this suffix pretty often when the arg is a byte slice) OTOH I think just `ImportKeys` is fine if we don't plan to import keys from other  stuff

**Comment by runcom** [comment]
Just a nit, LGTM otherwise, let me know if you're fine with my comment, otherwise again I'll merge

File: signature/mechanism.go (hunk 1 above)
Comment (runcom) [inline signature/mechanism.go]: much as you did here https://github.com/projectatomic/skopeo/pull/55/files#diff-230170c6339d5a6b9de551a0aeb6e74eR44

**Comment by mtrmac** [comment]
Good point, renamed.

**Pull request merged**